pkg/services/live/demultiplexer.go

```diff
@@ -0,0 +1,51 @@
+package live
+
+import (
+	"context"
+
+	"github.com/grafana/grafana-live-sdk/telemetry/telegraf"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana/pkg/models"
+)
+
+type Demultiplexer struct {
+	telegrafConverterWide *telegraf.Converter
+	managedStreamRunner   *ManagedStreamRunner
+}
+
+func NewDemultiplexer(managedStreamRunner *ManagedStreamRunner) *Demultiplexer {
+	return &Demultiplexer{
+		telegrafConverterWide: telegraf.NewConverter(),
+		managedStreamRunner:   managedStreamRunner,
+	}
+}
+
+func (s *Demultiplexer) GetHandlerForPath(_ string) (models.ChannelHandler, error) {
+	return s, nil
+}
+
+func (s *Demultiplexer) OnSubscribe(_ context.Context, _ *models.SignedInUser, e models.SubscribeEvent) (models.SubscribeReply, backend.SubscribeStreamStatus, error) {
+	reply := models.SubscribeReply{}
+	return reply, backend.SubscribeStreamStatusOK, nil
```

Comment:
should we throw an error for now?  I don't think we want to support/encourage subscribing to the push endpoint.  It is really a utility to match the HTTP endpoint

Reply:
done